topiary-cli/tests/samples/expected/ocaml.ml

```diff
@@ -1296,3 +1296,15 @@ type bar = {
   b: int;
   c: int;
 }
+
+(* #731 function dangling on multi-line calls *)
+let _ =
+  foo bar baz @@ fun x ->
+  x
+
+let _ =
+  foo
+    bar
+    baz
+    @@ fun x ->
+    x
```

Comment:
I think I'd have expected rather something like

```
    @@ fun x ->
      x
```

Any reason for the choice of not indenting the body further?

Reply:
Not an OCaml person, but I think the motivation is per [this comment](https://github.com/tweag/topiary/issues/731#issuecomment-2334077177) in #731:

> Yes, it is rather classic in OCaml to write something like
> 
> ```ocaml
> a_function @@ fun x ->
> another_function @@ fun y ->
> (x + y) / 2
> ```

i.e., I guess it's to avoid a Christmas Tree indentation situation.